docs/csharp/whats-new/csharp-9.md

```diff
@@ -0,0 +1,233 @@
+
+## Init only setters
+
+***Init only setters*** provide consistent syntax to initialize members of an object. Property initializers provide make it clear which value is setting which property. The downside is that those properties must be settable. Starting with C# 9.0, you can create `init` accessors, in addition of `set` accessors for properties and indexers. The main benefit is that callers can use property initializer syntax to set these values in creation expressions, but those properties are readonly once construction has completed. Init only setters provide a window to change state. That window closes when the construction phase ends. The construction phase effectively ends after all initialization, including property initializers and with-expressions have completed.
+
+The preceding example for positional records demonstrates using an init-only setter to set a property using a with expression. You can declare init only setters in any type you write. For example, the following struct defines a weather observation structure:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/WeatherObservation.cs" ID="DeclareWeatherObservation":::
+
+Callers can use property initializer syntax to set the values, while still preserving the immutability:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/WeatherObservation.cs" ID="UseWeatherObservation":::
+
+But, changing an observation after initialization is an error:
+
+```csharp
+// Error!
+now.TempetureInCelsius = 18;
+```
+
+In addition to using property initializers, init only setters can be useful to set base class properties from derived classes, or set derived properties through helpers in a base class. Positional records declare properties using init only setters. Those setters are used in with-expressions. You can declare init only setters for any `class` or `struct` you define.
+
+## Top-level statements
+
+***Top-level statements*** remove unnecessary ceremony from many applications. Consider the canonical "Hello World!" program:
+
+```csharp
+using System;
+
+namespace HelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+There’s only one line of code that does anything. With top-level statements, you can replace all that boilerplate with the `using` statement and the single line that does the work:
+
+:::code language="csharp" source="snippets/whats-new-csharp9/Program.cs" ID="TopLevelStatements":::
```

Comment:
I'm curious if we should truly express this as a single line, in other words drop the `using System;` and fully qualify `Console`?

```csharp
System.Console.WriteLine("Hello World!");
```